### DISCUSSION TITLE
Github Pages section not showing up in settings

### DISCUSSION THREAD

**Discussion by Xarcell** [community/community#29839] [Other Feature Feedback, Questions, & Ideas] [answered]
I'm trying to use Github to host a static site. I created a repo with my username: xarcell.github.io. I manually created a file index.html for testing purposes. When I click on settings, the Github Pages section that I see in documentation and YouTube videos do not appear. How do I fix this?

**Comment by airtower-luna** [comment, marked as answer]
That's hard to answer without seeing where you actually are looking (e.g. with a screenshot). However your repository  was deployed half an hour ago, so it looks like Pages are enabled. :wink:

**Reply by Xarcell** [reply]
Site is up, I guess it takes awhile. Still not seeing it in the repo settings though, somewhere above Danger Zone.

**Reply by airtower-luna** [reply]
Then I guess the documentation you have is outdated. Check on the left, there's a separate "Pages" link in the "Code and automation" section of the menu. :sweat_smile:

**Reply by yoannchaudet** [reply]
This ^

**Reply by dhoiffahmad** [reply]
I got the same problem, the github page in settings doesn't exist

Please help

**Comment by jfullstackdev** [comment]
this is all your content upon checking it by now 

![image](https://user-images.githubusercontent.com/47092464/185280797-c97fc9aa-eafd-4103-9e84-6b3adfdd3cff.png)

and of course the deployed page is 

![image](https://user-images.githubusercontent.com/47092464/185280870-6bc7521d-5513-4ad8-b0df-865c55319134.png)
